src/wallet/wallet.cpp

```diff
@@ -3150,7 +3162,62 @@ CWallet* CWallet::InitLoadWallet(bool fDisableWallet, const std::string& strWall
     }
     walletInstance->SetBroadcastTransactions(GetBoolArg("-walletbroadcast", DEFAULT_WALLETBROADCAST));
 
-    return walletInstance;
+    pwalletMain = walletInstance;
+    return true;
+}
+
+bool CWallet::ParameterInteraction()
```

Comment:
Never mind, it won't make much sense until you can explicitly pass it to GetArg(), like in https://github.com/bitcoin/bitcoin/pull/6423/commits/3d03f15126b79bd9f22ccd27f1d8ca43d9e16bf5#diff-b7702a084eb00fb47f9800fd68271951R315